python/cudf/cudf_pandas_tests/test_cudf_pandas.py

```diff
@@ -1421,3 +1423,24 @@ def test_holidays_within_dates(holiday, start, expected):
             utc.localize(xpd.Timestamp(start)),
         )
     ) == [utc.localize(dt) for dt in expected]
+
+
+def test_pandas_debugging_mode_option(monkeypatch):
+    import cudf.pandas
+    from cudf import Series
+
+    cudf.pandas.install()
+    import pandas as xpd
+
+    def mock_mean(self, *args, **kwargs):
+        return np.float64(1.0)
+
+    monkeypatch.setattr(Series, "mean", mock_mean)
+
+    with mock.patch.dict(os.environ, {"MODE_PANDAS_DEBUGGING": "True"}):
```

Comment:
Let's use `monkeypatch.setenv` instead. https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest.MonkeyPatch.setenv

Then we shouldn't need to import `from unittest import mock`.

```suggestion
    with monkeypatch.setenv("MODE_PANDAS_DEBUGGING", "True"):
```

Reply:
> Let's use `monkeypatch.setenv` instead. https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest.MonkeyPatch.setenv
It looks like it's not a context manager, but I applied a work around using `monkeypatch.context()`